packages/fluentui/react-northstar/src/components/Dropdown/DropdownSelectedItem.tsx

```diff
@@ -132,57 +135,63 @@ const DropdownSelectedItem: React.FC<WithAsProp<DropdownSelectedItemProps>> &
     },
   });
 
-  const contentElement = Box.create(header, {
+  const contentElement = Box.create(
+    Box.create(header, {
+      defaultProps: () => ({
+        as: 'span',
+        className: dropdownSelectedItemSlotClassNames.header,
+        styles: resolvedStyles.header,
+      }),
+    }),
+    {
+      defaultProps: () => ({
+        className: dropdownSelectedItemSlotClassNames.header,
+        styles: resolvedStyles.content,
+      }),
+    },
+  );
+
+  const iconElement = Box.create(icon, {
     defaultProps: () => ({
-      as: 'span',
-      className: dropdownSelectedItemSlotClassNames.header,
-      styles: resolvedStyles.header,
+      'aria-label': `Remove ${header} from selection.`, // TODO: Extract this in a behaviour.
+      className: dropdownSelectedItemSlotClassNames.icon,
+      styles: resolvedStyles.icon,
     }),
+    overrideProps: handleIconOverrides,
   });
 
-  const iconProps = _.isNil(icon)
-    ? icon
-    : {
-        name: null,
-        children: (ComponentType, props) =>
-          Box.create(icon, {
-            defaultProps: () => ({
-              'aria-label': `Remove ${header} from selection.`, // TODO: Extract this in a behaviour.
-              className: dropdownSelectedItemSlotClassNames.icon,
-              styles: resolvedStyles.icon,
-            }),
-            overrideProps: handleIconOverrides(props),
-          }),
-      };
-
-  const imageProps = _.isNil(image)
-    ? image
-    : {
-        children: (ComponentType, props) =>
-          Image.create(image, {
-            defaultProps: () => ({
-              avatar: true,
-              className: dropdownSelectedItemSlotClassNames.image,
-              styles: resolvedStyles.image,
-            }),
-            overrideProps: props,
+  const mediaElement = Box.create(
+    {},
+    {
+      defaultProps: () => ({
+        className: dropdownSelectedItemSlotClassNames.image,
+      }),
+      overrideProps: {
+        children: Image.create(image, {
+          defaultProps: () => ({
+            avatar: true,
+            className: dropdownSelectedItemSlotClassNames.image,
+            styles: resolvedStyles.image,
           }),
-      };
+        }),
+      },
+    },
+  );
 
   const element = (
     <Ref innerRef={itemRef}>
-      <Label
+      <ElementType
         className={classes.root}
         tabIndex={active ? 0 : -1}
-        styles={resolvedStyles.root}
-        circular
+        styles={resolvedStyles.main}
```

Comment:
Please why do you use `main` style here? And where is it defined, I don't see it in `dropdownSelectedItemStyles`

Reply:
Changed by @layershifter